### ISSUE TITLE
Anomaly not found in nametab (for syndef)

### ISSUE THREAD

**Issue by gares** [coq/coq#16922]
```coq
Module X.
Inductive t : Type := .
End X.
Notation t := X.t.
About t. (* anomaly *)
Print t. (* anomaly *)
```
This is on 387378141ed6c4426d4ebfee8f15766044909aba , while it works on 8.16.
CC 8.17 RM @Zimmi48

**Comment by gares** [comment]
cc @coq/parsing-maintainers

**Comment by Zimmi48** [comment]
ab52b254db8840c26a8d1a699ab9a00795d905f3 (#12324) is the first bad commit.

cc @herbelin

**Comment by herbelin** [comment]
Fixed in #16927. Thanks for the report.
